kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -653,6 +653,12 @@
         this.createSnackbar(this.questionsDeletedNotification$({ count }));
       },
     },
+    upLabel(){
+      return "up";
```

Comment:
Since this is not a function that returns a wrapped/translated string, this is going to fail the validation that you just added.

Reply:
I just forgot to update this :(